### PR TITLE
bsc#1036440 and bsc#994471: Command "yast remote list" should not alter the system.

### PR DESCRIPTION
**Pull request by gilsonsouza**

### CODE REVIEW

**Comment by lslezak** [comment]
The Travis fix is here: https://github.com/gilsonsouza/yast-network/pull/1

File: src/clients/remote.rb [hunk 1]
```diff
@@ -56,8 +56,6 @@ def main
         ),
         "id"         => "remote",
         "guihandler" => fun_ref(method(:RemoteGUI), "any ()"),
-        "initialize" => fun_ref(Remote.method(:Read), "boolean ()"),
-        "finish"     => fun_ref(Remote.method(:Write), "boolean ()"),
         "actions"    => {
           "list"  => {
```
Comment (lslezak) [inline src/clients/remote.rb]: Does the `list` command still work properly? The initialization has been moved from the global section only to the `allow` command. The question is do the other commands still work when the initialization is skipped?

Comment (gilsonsouza) [inline src/clients/remote.rb]: You're right... `list` cannot know the status of the system without initialization.
I'll fix this.

File: src/clients/remote.rb [hunk 2]
```diff
@@ -159,8 +158,9 @@ def SetRAHandler(options)
       else
         Remote.Disable
       end
+      success = Remote.Write
 
-      true
+      success
```
Comment (lslezak) [inline src/clients/remote.rb]: NP: You can avoid storing the result to `success` variable, you can directly return the `Remote.Write` value without a temporary variable.

**Pull request merged**